src/transforms/pipelines/mod.rs

```diff
@@ -43,6 +45,7 @@ impl PipelineConfig {
     }
 }
 
+/// This represent an ordered list of pipelines depending on the event type.
```

Comment:
This could use a bit more elaboration. From the description alone I'm still not sure what exactly it does.